jupyter-book/conditions/differential_gene_expression_environment.yml

```diff
@@ -0,0 +1,20 @@
+name: differential-gene-expression
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.9
```

Comment:
Should be

conda-forge::python=3.9.15

ideally

Reply:
ok, should I specify channels for other packages too or just python?

Reply:
Ideally for all packages, please

Always conda > pip and channel > no channel :)